presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveTableStatistics.java

```diff
@@ -687,9 +687,9 @@ public void testAnalyzeForTablePartitionedByBigint()
     {
         String tableNameInDatabase = mutableTablesState().get(NATION_PARTITIONED_BY_BIGINT_REGIONKEY.getName()).getNameInDatabase();
 
-        String showStatsWholeTable = "SHOW STATS FOR " + tableNameInDatabase;
-        String showStatsPartitionOne = "SHOW STATS FOR (SELECT * FROM " + tableNameInDatabase + " WHERE p_regionkey = 1)";
-        String showStatsPartitionTwo = "SHOW STATS FOR (SELECT * FROM " + tableNameInDatabase + " WHERE p_regionkey = 2)";
+        String showStatsWholeTable = format("SHOW STATS FOR %s", tableNameInDatabase);
+        String showStatsPartitionOne = format("SHOW STATS FOR (SELECT * FROM %s WHERE p_regionkey = 1)", tableNameInDatabase);
+        String showStatsPartitionTwo = format("SHOW STATS FOR (SELECT * FROM %s WHERE p_regionkey = 2)", tableNameInDatabase);
```

Comment:
please extract these changes as separate commit, it is not much related to the main change you are introducing. This comment applies also to other such changes in this file.